src/librustc_mir/borrow_check/nll/region_infer/error_reporting/mod.rs

```diff
@@ -370,6 +469,15 @@ impl<'tcx> RegionInferenceContext<'tcx> {
         diag.buffer(errors_buffer);
     }
 
+    /// Adds a suggestion to errors where a `impl Trait` is returned.
+    ///
+    /// ```text
+    /// help: to allow this impl Trait to capture borrowed data with lifetime `'1`, add `'_` as
```

Comment:
note to self: are we literally saying `'_` when we mean it as a placeholder for a concrete lifetime like `'a`? Or is it just supposed to be a replacement for the default of `'static` ?

Reply:
note to self and to @davidtwco : Wait, so **now** I don't get a "Submit review" prompt? what the hey...?